analyze-trace.ts

```diff
@@ -288,38 +342,46 @@ async function getTypes(): Promise<readonly any[]> {
     return typesCache!;
 }
 
-async function makePrintableTree(curr: EventSpan, currentFile: string | undefined, positionMap: PositionMap): Promise<{}> {
-    // Sort slow to fast
-    let childTree = {};
-
+async function makePrintableTree(curr: EventSpan, currentFile: string | undefined, positionMap: PositionMap): Promise<TreeNode | undefined> {
```

Comment:
"Printable" was my word for "treeify-compatible", so "makeTree" is probably fine now.